packages/eslint-plugin/src/configs/node-legacy.js

```diff
@@ -5,8 +5,8 @@ const configHelpers = require('../utils/configHelpers');
 
 /** @type {import("eslint").Linter.Config} */
 module.exports = {
-  extends: [path.join(__dirname, 'node')],
+  extends: [path.join(__dirname, 'core')],
   rules: {
-    ...configHelpers.getNamingConventionRule(true /* prefixWithI */),
+    ...configHelpers.getNamingConventionRule(true),
```

Comment:
```suggestion
    /* prefix interfaces with I */
    ...configHelpers.getNamingConventionRule(true),
```
might be worth keeping the comment ?
m